ydb/core/base/tablet_pipe.h

```diff
@@ -297,7 +309,7 @@ namespace NKikimr {
             // Creates and activated server, returns serverId.
             // Created server will forward messages to the specified recipent.
             virtual TActorId Accept(TEvTabletPipe::TEvConnect::TPtr &ev, TActorIdentity owner, TActorId recipient,
-                                    bool leader = true, ui64 generation = 0) = 0;
+                                    bool leader = true, ui64 generation = 0, const TString &versionInfo = {}) = 0;
```

Comment:
```suggestion
                                    bool leader = true, ui64 generation = 0, const TString& versionInfo = {}) = 0;
```

Во многих правках ниже смешался стиль.

https://docs.yandex-team.ru/arcadia-cpp/cpp_style_guide#ssylki-i-ukazateli
> Для ссылочных типов и указателей амперсанд (&) и звёздочка (*) приклеиваются к типу данных.

Reply:
Я тут просто придерживаюсь стиля ближайших строчек, к общему стилю лучше приводить отдельным pr'ом и когда-нибудь потом.